www/js/share/components/check-user-status/check-user-status.controller.js

```diff
@@ -257,6 +247,8 @@
         $scope.$on("get_account_status", (e, get_account_status) => {
             if (get_account_status.hasOwnProperty("status")) {
                 vm.status = get_account_status.status;
+                vm.promptClientToAuthenticate = get_account_status.prompt_client_to_authenticate;
```

Comment:
@nazaninreihani  It does not need to add the value to vm object if you don't need it more.